gallery/src/components/NxThreatBar/NxThreatBarByCategoryExampleListElements.tsx

```diff
@@ -10,37 +10,35 @@ import { NxThreatBar } from '@sonatype/react-shared-components';
 
 const NxThreatLevelsByNameListElements = () =>
   <>
-    <div className="nx-list">
-      <h4 className="nx-list__title">
-        List Title
-      </h4>
-      <ul>
-        <li className="nx-list__item">
-          <NxThreatBar />
-          <span>Threat Level Unspecified</span>
-        </li>
-        <li className="nx-list__item">
-          <NxThreatBar threatLevelCategory="none" />
-          <span>Threat Level None</span>
-        </li>
-        <li className="nx-list__item">
-          <NxThreatBar threatLevelCategory="low" />
-          <span>Threat Level Low</span>
-        </li>
-        <li className="nx-list__item">
-          <NxThreatBar threatLevelCategory="moderate" />
-          <span>Threat Level Moderate</span>
-        </li>
-        <li className="nx-list__item">
-          <NxThreatBar threatLevelCategory="severe" />
-          <span>Threat Level Severe</span>
-        </li>
-        <li className="nx-list__item">
-          <NxThreatBar threatLevelCategory="critical" />
-          <span>Threat Level Critical</span>
-        </li>
-      </ul>
-    </div>
+    <h3 className="nx-h3">
+      List Title
+    </h3>
+    <ul className="nx-list nx-list--threat-bars">
```

Comment:
As I mentioned in slack, the threat bars are going to be changing soon anyway (before the v2 feature branch goes to master), so I'm not going to check or review the changes that relate to them